zfsbootmenu/lib/zfsbootmenu-lib.sh

```diff
@@ -67,6 +67,51 @@ column_wrap() {
   echo -e "${footer//^/${pad}}"
 }
 
+# args: optional page to mark as active
+# prints: writes a string directly to the controlling terminal
+# returns: nothing
+
+# shellcheck disable=SC2120
+draw_page() {
```

Comment:
Would `tab_header` or `page_header` be more descriptive than `draw_page`?

Reply:
I changed it to `global_header` since it no longer actually draws directly to the screen.